src/backend/aspen/api/schemas/samples.py

```diff
@@ -61,6 +61,11 @@ def get(self, key: Any, default: Any = None) -> Any:
                 if self._obj.show_private_identifier
                 else None
             ),
+            "collection_location": (
```

Comment:
what if `collection_location` is an object like so:

```
"collection_location": {"region": "xxx", "country": "yyy", "division": "zzz", "location": null}
```

and we'll populate those fields with data from the locations table if we have a relationship to it, and we'll use the char fields otherwise?

Reply:
Well, the frontend wouldn't know how to handle it. It's a simple change, so I think that is best added as part of a future PR that updates how the samples table deals with locations.

Reply:
ok, fair enough